py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -440,6 +440,25 @@ def aten_ops_expand(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.amax.default)
```

Comment:
Based on [this schema](https://github.com/pytorch/pytorch/blob/1ef4bd169d4ea7729496e5734e87f26f8ff6f640/aten/src/ATen/native/native_functions.yaml#L3647-L3650), it seems that the dimension (`dim`) can also be not present. For instance, the following model creates the subsequent graph:
```python
class argmax(torch.nn.Module):
   def forward(self, x):
        return torch.argmax(x)

"""
graph():
    %arg0_1 : [num_users=1] = placeholder[target=arg0_1]
    %argmax : [num_users=1] = call_function[target=torch.ops.aten.argmax.default](args = (%arg0_1,), kwargs = {})
    return (argmax,)
"""
```
If we cannot support this case, you can add a `capability_validator` function to this decorator, which will note that case as unsupported. Roughly, that could be something like:

```python
def amax_param_validator(amax_node: Node) -> bool:
    return len(amax_node.args) >= 2

@dynamo_tensorrt_converter(
    torch.ops.aten.amax.default, capability_validator=amax_param_validator
)  # type: ignore[misc]
```

Reply:
@gs-olive Thank you so much George! That's a good catch. I fixed and learned a lot from these details recently :smiley:

Reply:
No problem - glad to hear it!